src/tangent.jl

```diff
@@ -202,15 +202,13 @@ end
 const TaylorBundle{N, B, P} = TangentBundle{N, B, TaylorTangent{P}}
 
 function TaylorBundle{N, B}(primal::B, coeffs) where {N, B}
-#    check_taylor_invariants(coeffs, primal, N) # TODO: renable this
+    check_taylor_invariants(coeffs, primal, N)
     _TangentBundle(Val{N}(), primal, TaylorTangent(coeffs))
 end
 
 function check_taylor_invariants(coeffs, primal, N)
     @assert length(coeffs) == N
-    if isa(primal, TangentBundle)
-        @assert isa(coeffs[1], TangentBundle)
```

Comment:
this check is just wrong AFAICT.
because if the primal is a TangentBundle, then the first derivative is a `Tangent{TangentBundle}`
The first deriviative has no primal component.

But this was never hit before because we were not making `TaylorBundles` when taking derivatives of TaylorBundles, (at least not in anything that happens in our `test/forward.jl` I checked)  we were (I assume) making `CompositeBundles`